services/meta/client.go

```diff
@@ -1251,6 +1258,14 @@ func (e errRedirect) Error() string {
 	return fmt.Sprintf("redirect to %s", e.host)
 }
 
+type errCommand struct {
```

Comment:
No particular reason other than the error above did it this way: https://github.com/influxdata/influxdb/pull/5706#discussion-diff-53258477R1253